client/src/Components/reviews/AddReview.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect } from 'react';
+import OverallRating from './OverallRating.jsx';
+import HelpfulRadio from './HelpfulRadio.jsx';
+import Comfort from './Comfort.jsx';
+import Fit from './Fit.jsx';
+import Length from './Length.jsx';
+import Quality from './Quality.jsx';
+
+function AddReview(props) {
+  const [isHelpful, setIsHelpful] = useState('');
+  const [starRating, setStarRating] = useState(0);
+  const [comfort, setComfort] = useState(0);
+  const [fit, setFit] = useState(0);
+  const [length, setLength] = useState(0);
+  const [quality, setQuality] = useState(0);
+  const [charMin, setCharMin] = useState(50);
+  const [review, setReview] = useState('');
+  const [summary, setSummary] = useState('');
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+
+  const helpful = (boolean) => {
+    setIsHelpful(boolean);
+    console.log(isHelpful);
+  };
+
+  const handleStarRating = (number) => {
```

Comment:
all of these 'handle' functions are unnecessary. You can pass the state setter function directly to the children components.